tests/unit/checkboxradio/checkboxradio_methods.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * checkboxradio_methods.js
+ */
+
+
+(function($) {
+
+module( "Checkboxradio: methods" );
+	test( "Checkbox: refresh", function() {
```

Comment:
Bad indent of the whole while, also drop the subject for all tests, here it should be just "refresh".